src/style.css

```diff
@@ -1 +1,135 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 300;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 600;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+form {
+  display: block;
+  unicode-bidi: isolate;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', sans-serif;
+  font-size: 16px;
+}
+
+label {
+  cursor: default;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
+.search-form__label:after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
+.search-form__label_big:after {
+  background-image: url('../src/images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
+.search-form__label_small:after {
+  background-image: url('../src/images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search-form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form__input_big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 50px;
+}
+
+.search-form__input_small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.search-form__input_big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__input_small::placeholder {
+  font-size: 14px;
+  line-height: 22px;
```

Comment:
This style is doublicated in both placeholder selectors. please remove the doublication.